qemu/tests/live_snapshot_runtime.py

```diff
@@ -0,0 +1,65 @@
+import time
+import random
+
+from autotest.client.shared import error
+from autotest.client.shared import utils
+from qemu.tests import live_snapshot_basic
+
+
+class LiveSnapshotRuntime(live_snapshot_basic.LiveSnapshot):
+
+    process = []
+
+    def __init__(self, test, params, env, tag):
+        super(LiveSnapshotRuntime, self).__init__(test, params, env, tag)
+
+    @error.context_aware
+    def reboot(self):
+        """
+        Reset guest with system_reset;
+        """
+        method = self.params.get("reboot_method", "system_reset")
+        return super(LiveSnapshotRuntime, self).reboot(method=method)
+
+    def action_before_start(self):
```

Comment:
How about add function this to block_copy.py as something like action_when_jobdoing? I think this could be shared by all block jobs.